### PR TITLE
DOCSP-30630 Navigate Diagrams and Entities Drawer

### PR DESCRIPTION
**Pull request by ianf-mongodb**
## Description

Navigate Diagrams and Entities Drawer conversion from markdown to snooty.
Tabbed the images so they do not take up as much page real estate, also lightly reworded page content.

## Staging

https://docs-mongodbcom-staging.corp.mongodb.com/docs-relational-migrator/docsworker-xlarge/DOCSP-30630/diagrams/navigate-diagrams/navigate-diagrams/

https://docs-mongodbcom-staging.corp.mongodb.com/docs-relational-migrator/docsworker-xlarge/DOCSP-30630/diagrams/navigate-diagrams/apply-layout/

## Jira

https://jira.mongodb.org/browse/DOCSP-30630

## Build

https://workerpool-boxgs.mongodbstitch.com/pages/job.html?collName=queue&jobId=6489cb1d67a273b81627cd9a

### CODE REVIEW

**Comment by ianf-mongodb** [comment]
@sarah-olson-mongodb Wanted to get this in review queue before I worked on the landing page.

File: source/diagrams/navigate-diagrams/apply-layout.txt
Comment (sarah-olson-mongodb) [inline source/diagrams/navigate-diagrams/apply-layout.txt:33]: (praise) 
Absolutely fantastic use of tabs. Nice work @ianf-mongodb.

**Review by sarah-olson-mongodb — APPROVED** [review]
Great work @ianf-mongodb! LGTM after a few super minor nits.

File: source/diagrams/navigate-diagrams/navigate-diagrams.txt
Comment (ianf-mongodb) [reply source/diagrams/navigate-diagrams/navigate-diagrams.txt:22]: Opting for Change the Active Diagram here.

**Pull request merged**
